plugins/adobe-analytics-response-events/index.ts

```diff
@@ -53,6 +53,24 @@ import { ValidationPluginResult } from '../../types/validationPlugin';
       notMatchedEvents.push(analyticsTrackEvent.uuid);
     }
   }
+
+  if (notMatchedEvents.length) {
+    const configurationEvents: Configuration[] = match(
+      aepMobile.configuration.matcher,
+      events
+    );
+    const optedin = configurationEvents.some((event) => {
+      const eventData = aepMobile.configuration.getEventData(event);
+      const privacy = eventData['global.privacy'];
+      return privacy === 'optedin';
+    });
+
+    if (!optedin) {
+      notMatchedMessage +=
+        ' If your report suite is not timestamp enabled, hits are discarded until the privacy status changes to `optedin`';
```

Comment:
What happens if it is timestamp enabled? Do the hits get queued, but not sent?

Reply:
This is a place where a doc link will be really nice. What's this `privacy status` and how do I change it to `optedin`